tests/modules/teams/resources/test_options.py

```diff
@@ -0,0 +1,31 @@
+# pylint: disable=missing-docstring
+import pytest
+
+@pytest.mark.parametrize('path,status_code', (
+    ('/api/v1/teams/', 401),
+    ('/api/v1/users/1', 401),
+))
+def test_teams_options_unauthorized(path, status_code, flask_app_client):
+    response = flask_app_client.options(path)
+
+    assert response.status_code == status_code
+
+
+@pytest.mark.parametrize('path,result', (
+    ('/api/v1/teams/', ['POST', 'OPTIONS']),
+    ('/api/v1/teams/1', ['GET', 'OPTIONS', 'PATCH']),
+    ('/api/v1/teams/2', ['OPTIONS']),
+))
+def test_teams_options_authorized(
+        path,
+        result,
+        flask_app_client,
+        regular_user,
+        team_for_regular_user,
+        team_for_nobody
+    ):
+    with flask_app_client.login(regular_user, auth_scopes=('teams:write', 'teams:read')):
+        response = flask_app_client.options(path)
+
+    assert response.status_code == 204
+    assert not (set(result) - set(response.headers['Allow'].split(', ')))
```

Comment:
Oh, actually, the better way is to compare sets because you don't want the test pass when the expected result is `['OPTIONS']`, and returned value is `['OPTIONS', 'GET']`. So:

```python
assert set(result) == set(response.headers['Allow'].split(', '))
```

I would also rename `result` into `expected_allowed_methods`, and define them as sets from the beginning.